CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Development
+
+- Use Typescript 4.0.3
+- Adapted a lot of dependecies to align with lovely-ra
```

Comment:
Does https://github.com/lovelysystems/lovely-ra/pull/66#discussion_r502284245 fix the changes here too?

Reply:
probably yes, I'll add the config in this project too, as it is still missing, and I also have to fix the tests here, as they fail on the individual checkout.